docs/README.android

```diff
@@ -55,64 +58,41 @@ To develop Kodi for Android the Android SDK and NDK are required.
 --------------------------------------------------------------------
 3.1. Prerequisites when compiling on Mac OSX
 --------------------------------------------------------------------
-
-a. When building on Mac OSX you should download the java version from here: http://support.apple.com/kb/DL1572
-Or ensure otherwise that you are using java 1.6 (java -version tells you the version).
-b. The compilation for android needs a case sensitive filesystem. The
-   filesystem used on normal OSX installations is case insensitive. So you
-   need to generate a writeable hdd image and format it with hfs+ (case sensitive).
-   The size should be 20GB.
-c. Generate a writable dmg with the following command:
-    $ hdiutil create -type UDIF -fs 'Case-sensitive Journaled HFS+' -size 20g -volname android-dev ~/android-dev.dmg
-
-d. Whenever you want to compile/develop you need to mount it:
-    $ open ~/android-dev.dmg
-
-e. Your workspace can be accessed in /Volumes/android-dev/ then
-f. Once you have your hdd image with case sensitive hfs+ file system do all steps
-   below inside of this filesystem. In the end you need to adapt all paths in this
-   document so that they match your local environment. As an example here is a
-   configure line from step 5.1 which demonstrates possible paths. In this
-   example the hdd image is mounted on /Volumes/android-dev.
-    $ ./configure --with-tarballs=/Users/Shared/xbmc-depends/tarballs --host=arm-linux-androideabi \
-                  --with-sdk-path=/Volumes/android-dev/android/android-sdk-macosx \
-                  --with-ndk=/Volumes/android-dev/android/android-ndk-r12b \
-                  --with-toolchain=/Volumes/android-dev/android/android-toolchain-arm/android-21 \
-                  --prefix=/Volumes/android-dev/android/xbmc-depends
```

Comment:
Well i used to.Didn't try it for quiet some time but the general process in doing so should still be valid.